src/pudl/output/ferc1.py

```diff
@@ -3036,6 +3089,7 @@ def out_ferc1__yearly_rate_base(
         .assign(
             dollar_value=lambda x: x.dollar_value.divide(8),
```

Comment:
Could we just name this `ending_balance` here so we don't have to use `.rename(columns={"dollar_value": "ending_balance"}` a few lines down?

Reply:
we could but then we'd have two $ columns in the data with very different values. we'd need to drop `dollar_value` below. is this blocking?

Reply:
no